README.md

```diff
@@ -1,145 +1,47 @@
+1. Select one of the offices
+2. Select a year between 2016 - 2024 (inclusive)
+3. Show which fruit was eaten most (in that year for that office)
+4. Show how much fruit was eaten on average per person (in that year for that office)
 
-Once more, feel free to use whatever libraries (UI or otherwise) you want, you will not be judged by the size of your node_modules directory nor your artistic abilities.
+### 2. Purchase fruit
 
-We would however prefer you use React as a basis for the frontend, if that is an option for you.
+1. Select one of the offices
+2. Select a fruit and the quantity to purchase
+3. Prevent the purchase if the total calories of the purchase exceed 1000kcal
+4. Persist the purchase in the database
 
-### 2. API
+## Constraints
 
-We envision you will probably need two endpoints to satisfy the Frontend requirements ( maybe ```/purchase``` and  ```/report``` ).
+1. Use Typescript / Javascript
+2. Front-end preferably in React
+3. No restrictions on third party libraries
+4. Suggested backend server: Express or Koa
 
-Having said that, you do have the full freedom to organize how many endpoints you need and however you want to organize them, as long as the frontend requirements are satisfied.
+## Resources
 
-You are free to use any out of the box server or any libraries you want; we'd suggest `express` or `koa`.
+1. [Fruityvice](https://www.fruityvice.com/) to get info on certain fruits
+2. Postgres database provided with this task (`./init.sql`)
 
-You DO NOT need to provide endpoints to get a list of office locations or available fruits. If you don't have the time or energy feel free to hardcode these lists, this will not be taken into consideration. The way you do hardcode them, will.
+## How to run
 
-### 3. A nice way to run your solution
-
-Please, **provide a concise one or two line command to start your project**.
-If any explanation is necessary please create a HOWTO.md and explain how to run your solution.
-
-### 4. Answers to technical questions
-
-Please answer the following questions in a markdown file called **"ANSWERS.md"**.
-
-1. How long did you spend on the coding test?
-
-2. What would you add to your solution if you had more time? If you didn't spend much time on the coding test then use this as an opportunity to explain what you would add.
-
-3. Describe your solution in plain english. Point out your design decisions and the reasoning behind them.
-
-4. Have you learned something new doing this test? If yes, describe it here.
-
-5. Describe yourself using JSON.
-
-## Final tips
-
-- Feel free to ask for clarification if something is unclear in the task
-- Parts of the database records are generated randomly, don't expect consistancy if you recreate the container volumes.
-- Try not to over-engineer. "Everything should be made as simple as possible, but not simpler."
+```shell
+npm run setup
+npm run dev
```

Comment:
This does not work out of the box. `setup` command is missing `api` part of the code.